ui/src/app/views/concept-homepage/component.ts

```diff
@@ -312,4 +347,14 @@ export class ConceptHomepageComponent implements OnInit {
   domainLoading(domain) {
     return this.searchLoading || !this.completedDomainSearches.includes(domain.domain);
   }
+
+  filterConceptSelection(concepts) {
+    concepts.forEach((concept) => {
+      const exist = this.selectConceptMap.get(concept.conceptId);
+      if (exist) {
+        concept.selected = true;
```

Comment:
It has been set to False by default in both the instance its called.

Reply:
I saw that - I was suggesting it might be cleaner to handle both things here rather than using a two-step process. I may be missing something else though